content/en/coterm/_index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Record terminal sessions and processes with CoTerm
+---
+
+CoTerm enables you to record terminal sessions and analyze these recordings in Datadog. Investigating terminal sessions provides context about how system and security incidents were caused and remediated.
+
+For your security, CoTerm uses [Sensitive Data Scanner][2] to detect and obfuscate sensitive data, such as passwords and API keys.
+
+## Setup
+
+1. Install CoTerm:
+
+   ```shell
+   curl --tlsv1.2 --proto '=https' -sSf 'https://update.coscreen.org/install-ddcoterm.sh' | bash
+   ```
+
+2. Set your [Datadog API key][1] as an environment variable:
+
+   ```shell
+   export DD_API_KEY=<DATADOG_API_KEY>
+   ```
+   Replace `<DATADOG_API_KEY>` with your API key.
```

Comment:
```suggestion
2. Initialize and authenticate CoTerm:

   ```shell
   ddcoterm init
   ```
```

These days it's easiest to use `ddcoterm init` to set up/authenticate CoTerm.